### PR TITLE
Refactor server track model

### PR DESCRIPTION
**Pull request by denysgud**

### CODE REVIEW

File: client/components/place/placesOnMap.factory.js [hunk 1]
```diff
@@ -87,7 +87,16 @@ angular.module('mapModule')
...
     var addTrack = function(track) {
       var color = mapMarkingTypes.tracks[track.type].color;
-      trackForAdding = polyline(track.location.coordinates, color).addTo(map);
+      var coordsArray = [];
+      track.places.forEach(function(place, index) {
+        var coords = [];
+        coords[0] = place.location.coordinates[1];
+        coords[1] = place.location.coordinates[0];
+        coordsArray[index] = coords;
+        console.log(coordsArray);
```
Comment (max-yushchuk) [inline client/components/place/placesOnMap.factory.js]: Remove log

File: client/components/place/placesOnMap.factory.js [hunk 2]
```diff
@@ -87,7 +87,16 @@ angular.module('mapModule')
...
+      });
+      trackForAdding = polyline(coordsArray, color).addTo(map);
+        console.log(trackForAdding);
```
Comment (max-yushchuk) [inline client/components/place/placesOnMap.factory.js]: And here remove log

File: client/components/place/placesOnMap.factory.js (hunk 1 above)
Comment (denysgud) [inline client/components/place/placesOnMap.factory.js]: Already)

File: server/app/models/track.js [hunk 3]
```diff
@@ -27,13 +16,28 @@ var TrackSchema = new Schema({
...
     required: true
   },
-  location: locSchema,
+  places: [{
+    type: Schema.Types.ObjectId,
+    ref: 'Place',
+    required: true
+  }],
   description: String,
   trackRate: Number,
```
Comment (max-yushchuk) [inline server/app/models/track.js]: trackRate should count by number of likes. But I think, it'll be in future. Now it is good for me :)

Comment (yurykozhenov) [inline server/app/models/track.js]: Rename to just `rate`

**Pull request merged**